python/ql/src/semmle/python/dataflow/new/internal/DataFlowPublic.qll

```diff
@@ -266,7 +266,10 @@ ParameterNode parameterNode(Parameter p) { result.getParameter() = p }
 
 /** A data flow node that represents a call argument. */
 class ArgumentNode extends Node {
-  ArgumentNode() { this = any(DataFlowCall c).getArg(_) }
+  ArgumentNode() {
+    this = any(DataFlowCall c).getArg(_) and
+    this.(CfgNode).getNode() = any(CallNode c).getArg(_)
```

Comment:
I think this needs to be an `or`, otherwise we don't expand the `ArgumentNode` class :sweat_smile: 

```suggestion
    this = any(DataFlowCall c).getArg(_)
    or
    this.(CfgNode).getNode() = any(CallNode c).getArg(_)
```

Reply:
Ha! Good point :-)